prog/vm/nexus.rb

```diff
@@ -428,6 +432,9 @@ def write_params_json
       hop_prevent_destroy
     end
 
+    when_waiting_for_capacity_set? do
```

Comment:
We already clean up semaphores while destroying the strand, so there's no need to manually decrease it here.

Reply:
I see, you did it because semaphore check has more precedence than the column value

Reply:
We might change "deleting" state similar to the PG resource

```ruby
return "deleting" if destroy_set? || strand.label == "destroy"
```